gedcom_view.py

```diff
@@ -24,13 +24,14 @@ def green(s):
 
 if len(sys.argv) == 1:
     gedcom = sys.stdin
+    buffer = gedcom.read().replace('\r\n', '\n').replace('\r', '\n')
```

Comment:
Perhaps we no longer need the `replace` here... I had done that years ago, but python3 and `readlines` are in fact likely to already do this normalization

Reply:
Yes. Since v 3.0, python defaults to `Universal Newline` unless otherwise specified. I left that change until I could read up on stdin behavior, but have now removed the `replace`.